inc/alert.class.php

```diff
@@ -155,6 +160,7 @@ static function dropdownYesNo($options = []) {
     */
    static function dropdownIntegerNever($name, $value, $options = []) {
 
+      $p = [];
```

Comment:
```php
$p = [
   'min' => 1,
   'max' => 100,
   'step' => 1
];
```
Would be better

Reply:
I did not do that to make changes easier, but I agree, it would be better. I will do it.